plugins/fullscreen.md

```diff
@@ -12,6 +12,11 @@ controls: toolbar button, menu item
 
 This plugin adds full screen editing capabilities to {{site.productname}}. When the toolbar button is pressed the editable area will fill the browser's viewport. The plugin adds a toolbar button and a menu item `Fullscreen` under the `View` menu.
 
+Full screen mode can be toggled using the following keyboard shortcuts:
+
+* MacOS: Command+Shift+F
```

Comment:
This should probably be `macOS` to match the actual OS name.

Reply:
```suggestion
* macOS: Command+Shift+F
```